### ISSUE TITLE
execa `$` doesn't work with `stdio: inherit`

### ISSUE THREAD

**Issue by jcbhmr** [oven-sh/bun#7068]
### What version of Bun is running?

1.0.11+f7f6233ea

### What platform is your computer?

Linux 5.15.90.1-microsoft-standard-WSL2 x86_64 x86_64

### What steps can reproduce the bug?

```js
import { $ } from "execa"
const args = ["hello", "world"]
await $({ stdio: "inherit" })`echo ${args}`
```
```sh
bun index.js
```

### What is the expected behavior?

```sh
hello world
```

### What do you see instead?

```sh
hello world
1 | export const getStreamContents = async (stream, {init, convertChunk, getSize, truncateChunk, addChunk, getFinalChunk, finalize}, {maxBuffer = Number.POSITIVE_INFINITY} = {}) => {
2 |     if (!isAsyncIterable(stream)) {
3 |             throw new Error('The first argument must be a Readable, a ReadableStream, or an async iterable.');
            ^
error: Command failed: echo hello world
The first argument must be a Readable, a ReadableStream, or an async iterable.
      at /home/jcbhmr/.bun/install/cache/get-stream@8.0.1/source/contents.js:3:8
      at getStreamContents (/home/jcbhmr/.bun/install/cache/get-stream@8.0.1/source/contents.js:1:40)
      at getStreamAsString (/home/jcbhmr/.bun/install/cache/get-stream@8.0.1/source/string.js:4:40)
      at /home/jcbhmr/.bun/install/cache/execa@8.0.1/lib/stream.js:119:23
      at getSpawnedResult (/home/jcbhmr/.bun/install/cache/execa@8.0.1/lib/stream.js:118:32)
      at /home/jcbhmr/.bun/install/cache/execa@8.0.1/index.js:118:93
      at /home/jcbhmr/.bun/install/cache/execa@8.0.1/index.js:117:23
      at onetime (/home/jcbhmr/.bun/install/cache/onetime@6.0.0/index.js:18:17)
      at /home/jcbhmr/.bun/install/cache/execa@8.0.1/lib/promise.js:15:34
      at processTicksAndRejections (:61:76)
```

### Additional information

this also doesn't work:
```js
import { $ } from "execa"
const arg = "hello world"
await $({ stdio: "inherit" })`echo ${arg}`
```
```sh
hello world
error: Command failed: echo hello world
The first argument must be a Readable, a ReadableStream, or an async iterable.
```

but this does work:
```js
import { $ } from "execa"
const args = ["hello", "world"]
const { stdout } = await $`echo ${args}`
console.log(stdout)
```
```sh
hello world
```

as does this
```js
import { $ } from "execa"
const arg = "hello world"
const { stdout } = await $`echo ${arg}`
console.log(stdout)
```
```sh
hello world
```

**Comment by elie222** [comment]
Ran into a similar issue. `npx tsx` worked. `bun` didn't. i was also getting silent errors with bun.

**Comment by ftzi** [comment]
execaCommand and execa have the same issue. https://bun.sh/docs/api/spawn can be an alternative in the meantime.

Edit: Tbh, I even preferred the bun's spawn as it's one dep less (two, including @types) to be installed. It could support string as a param instead of an array, like execaCommand does.

**Comment by Jarred-Sumner** [comment]
Fixed in Bun v1.1.25
